external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/DB2IntegrationSuite.scala

```diff
@@ -30,7 +30,7 @@ import org.apache.spark.tags.DockerTest
  * To run this test suite for a specific version (e.g., ibmcom/db2:11.5.4.0):
  * {{{
  *   DB2_DOCKER_IMAGE_NAME=ibmcom/db2:11.5.4.0
- *     ./build/sbt -Pdocker-integration-tests "test-only *DB2IntegrationSuite"
+ *     ./build/sbt -Pdocker-integration-tests "testOnly *v2.DB2IntegrationSuite"
```

Comment:
oh BTW, I think `testOnly` doesn't work anymore after we upgraded SBT. It should be `test-only`, see also https://github.com/apache/spark/pull/30028

Reply:
`test-only` -> `testOnly`? `test-only` doesn't work any more?

Reply:
oops

Reply:
:)